platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#Due to the hardware design, as4630-54pe use "eth2" instead of "eth0" as management interface.
+#Rename netdev "eth0" and "eth2" to swap original "eth2" to "eth0".
```

Comment:
better to use u-dev rules to name the management interface.

Reply:
Dear lguohan:
Thank you for your suggestion, and the method of udev is work will after the testing.
But with the current code base，I need to add the rules for udev to create the path about /etc/udev/rules.d in "rules"(common code.)
It’s a bit difficult to modify the common place in accton project. So I need this commit to merge into this project.

BR.
Thank you